prover/prover_dal/src/fri_gpu_prover_queue_dal.rs

```diff
@@ -187,14 +197,25 @@ impl FriGpuProverQueueDal<'_, '_> {
         let prover_max_age =
             pg_interval_from_duration(Duration::from_secs(archive_prover_after_secs));
 
+        // TODO: probably there is a way to make this query better
```

Comment:
Initially I didn't need to specify names of fields, but since I've added new column after archived_at, the ordering matters now and we need to specify every field that is retrieved from the delete, which I don't really like, but I didn't find a better way for now